src/utils/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin set-flex-property {
+  justify-content: space-between;
+  align-items: center;
+}
+
+@mixin buy-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 700;
+  font-size: 14px;
+  height: 40px;
+  line-height: 16px;
+  border-radius: 5px;
+}
```

Comment:
do you really need this mixin? By its name and current usage I don't think this is reusable one

<img width="742" alt="image" src="https://user-images.githubusercontent.com/103033758/235708042-a83752a5-efb7-49f4-ac16-520211eabdb6.png">